packages/repository/src/repositories/repository.ts

```diff
@@ -37,6 +38,22 @@ export interface ExecutableRepository<T extends Model> extends Repository<T> {
   ): Promise<AnyObject>;
 }
 
+export type TransactionalEntityRepository<
+  T extends Entity,
+  ID,
+  Relations extends object = {}
+> = TransactionalRepository<T> & EntityCrudRepository<T, ID>;
+export interface TransactionalRepository<T extends Entity>
+  extends Repository<T> {
+  /**
+   * Begin a new Transaction
+   * @param options - Options for the operations
+   * @returns Promise<true> if an entity exists for the id, otherwise
```

Comment:
The docs says that the functions returns a promise of a boolean, but the actual code uses `Promise<Transaction>`. Please fix the docs.